src/test/java/trello/ui/main.java

```diff
@@ -5,41 +5,84 @@
-//        System.setProperty("webdriver.chrome.driver", "d:\\otros\\webUI\\chromedriver_win32\\chromedriver.exe");
+
         //Configure web driver manager
         WebDriverManager.chromedriver().setup();
 
-        //Create web driver
-        //Open browser
-        WebDriver driver = new ChromeDriver();
-        //Navigate
-//        driver.get("https://www.ebay.com/");
-        driver.navigate().to("https://www.ebay.com/");
-        //Close browser
-//        driver.close();
-        //Find element
-        WebElement searchBox = driver.findElement(By.cssSelector("#gh-ac"));
-        searchBox.sendKeys("Cellphones");
-        searchBox.clear();
-        searchBox.sendKeys("Laptops");
-        //Click search
-        WebElement searchButton = driver.findElement(By.cssSelector("#gh-btn"));
-        searchButton.click();
-        //Close browser and kill process
+        //Create web driver. Open browser
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--lang=en");
+        WebDriver driver = new ChromeDriver(options);
+        driver.manage().window().maximize();
 
-        //Get list of elements
-        List<WebElement> linksList = driver.findElements(By.cssSelector("#gh-topl>li"));
-        for (WebElement link: linksList) {
-            System.out.println(link.getText());
+        //1.- Navigate to Trello home page
+        driver.navigate().to("https://trello.com/");
+
+        //2.- Print in Cosole Main message
+        WebElement welcomeMesssage = driver.findElement(By.cssSelector(".col-lg-6>h1"));
+        System.out.println(welcomeMesssage.getText());
+
+        //3.- Click Log in button
+        WebElement loginHomeButton = driver.findElement(By.cssSelector("[href*='login']"));
+        loginHomeButton.click();
+        waitForPageLoaded(driver, 1000);
+
+        //4.- Fill username
+//        WebElement userInput = driver.findElement(By.cssSelector("#user[name='user']"));
+        WebElement userInput = driver.findElement(By.cssSelector("#user"));
+        userInput.sendKeys("art_ag@hotmail.com");
+        //Click on Log in with Atlassian
+//        WebElement loginAtlassian = driver.findElement(By.cssSelector(".button#login"));
+        WebElement loginAtlassian = driver.findElement(By.cssSelector("#login"));
+        loginAtlassian.click();
+        waitForPageLoaded(driver, 1000);
+
+        //5.- Fill password
+        WebElement passwordInput = driver.findElement(By.cssSelector("#password"));
+        passwordInput.sendKeys("12345678");
+
+        //6.- Click on Login button
+        WebElement loginButton = driver.findElement(By.cssSelector("#login-submit"));
+        loginButton.click();
+        waitForPageLoaded(driver, 5000);
+
+        //7.- Get all element is sidebar
+//        WebElement sibeBar = driver.findElement(By.cssSelector("._27QKbGhxDkdNHq>ul li"));
+        List<WebElement> sibeBarList = driver.findElements(By.cssSelector("._27QKbGhxDkdNHq>ul li"));
```

Comment:
@ArturoAmoroso my suggestion here is try to avoid complex value classes for example something more friendly like 
//*[@class='home-left-sidebar-container']/div/ul/descendant::li
Good exercise trying to reach those three specific items.